model-optimizer/install_prerequisites/install_prerequisites.sh

```diff
@@ -181,21 +166,20 @@ find_ie_bindings() {
 }
 
 if [[ $V_ENV -eq 1 ]]; then
-    "$python_binary" -m venv "$SCRIPTDIR/../venv${postfix}"
-    source "$SCRIPTDIR/../venv${postfix}/bin/activate"
-    venv_python_binary="$SCRIPTDIR/../venv${postfix}/bin/$python_binary"
+    "$python_binary" -m venv "$HOME/venv_mo${postfix}"
```

Comment:
Should we have a single venv directory for all OpenVINO tools? If so then we may need to rename the directory to "venv_openvino" or "venv_ov".